lib/chooseTeam.js

```diff
@@ -0,0 +1,43 @@
+const rl = require("./rl");
+const logGrid = require("./logGrid");
+const checkWin = require("./checkWin");
+
+const getTeamChoice = () => {
+  return new Promise((resolve) => {
+    rl.question("Pick your team. O or X? = ", (choice) => {
+      return resolve(choice.toUpperCase());
+    });
+  });
+};
+
+const getValidTeamChoice = (choice) => {
+  if (choice === "O" || choice === "X") {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const setTeam = (store, choice) => {
+  store.userTeam = choice;
+  if (choice === "O") {
+    store.cpuTeam = "X";
+  } else {
+    store.cpuTeam = "O";
+  }
+};
+
+const storeTeamChoice = async (store) => {
+  let choice = await getTeamChoice();
+  if (getValidTeamChoice(choice) === true) {
+    setTeam(store, choice);
```

Comment:
after above refactors, there is no need for an if statement here, because choice will always be valid